src/flux/task-step.coffee

```diff
@@ -51,6 +51,14 @@ TaskStepConfig =
           isAnswered = false
       isAnswered
 
+    isCore: (id) ->
+      step = @_get(id)
+      coreGroups = [
+        'core'
+        'default'
```

Comment:
I'm not sure when, but the BE sometimes returns it as 'default' and it is defined in the [napkin notes](https://github.com/openstax/napkin-notes/blob/master/jp/exercise_sources.md) as well

@kevinburleigh75 i think has insight into this